src/pages/repo-new.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import t from 'tcomb-form'
+import _ from 'lodash'
+
+let Form = t.form.Form;
+
+let Repo = t.struct({
+  name: t.Str,
+});
+
+export default React.createClass({
+
+  getInitialState() {
+    return {
+      options: {
+        hasError: false,
+        error: <i> Important Error!</i>,
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected '}' to match '{' from line 15 and instead saw 'i'.<br>Expected '}' to match '{' from line 14 and instead saw 'Error'.<br>Expected an operator and instead saw '!'.<br>Unclosed regular expression.<br>Unrecoverable syntax error. (22% scanned).